src/index.js

```diff
@@ -244,7 +244,40 @@ async function getHtsStorageAt(address, requestedSlot, blockNumber, mirrorNodeCl
                 ZERO_HEX_32_BYTE,
                 `Failed to get the metadata of the NFT ${tokenId}#${serialId}`
             );
-        persistentStorage.store(tokenId, blockNumber, nrequestedSlot, atob(metadata));
+        persistentStorage.store(
+            tokenId,
+            blockNumber,
+            nrequestedSlot,
+            atob(metadata),
```

Comment:
Let's use `Buffer.from(metadata, 'base64').toString()` to be consistent with similar code below.